cleversonledur/doctalk-pdf-langchain-rag
# Add initial code for doctalk

## Summary

This PR adds the initial codebase for the Doctalk project. 

## Changes

- Added `document`, `text`, and `vector` modules.
- The `document` module contains the `loader.py` script for loading documents.
- The `text` module contains the `splitter.py` script for splitting text.
- The `vector` module contains the `store.py` script for creating a vector store.
- The `main.py` script ties everything together and provides the user interface for the system.

## Testing

To test the changes, follow these steps:

1. Clone the repository and checkout to this branch.
2. Install the required packages using `pip install -r requirements.txt`.
3. Run the main script using `python main.py`.
4. When prompted, enter a question to see the system's response.

## Next Steps

After this PR is merged, the next steps will be to improve the text splitting and document loading functionalities, and to add more comprehensive tests for the system.
